ktor-client/ktor-client-core/common/src/io/ktor/client/utils/CoroutineUtils.kt

```diff
@@ -8,7 +8,7 @@ import io.ktor.utils.io.*
 import kotlinx.coroutines.*
 
 /**
- * Creates [CoroutineDispatcher] for client with fixed [threadCount] and specified [dispatcherName].
+ * Creates [CoroutineDispatcher] for the client with fixed [threadCount] and specified [dispatcherName].
```

Comment:
Here we can also use `@param` to describe `threadCount` and `dispatcherName` in the same way it is done in the other KDocs.